include/aie/Dialect/AIEVec/IR/AIEVecOps.td

```diff
@@ -727,4 +728,27 @@ def AIEVec_ExtElemOp:
   let assemblyFormat = "$source `,` $index attr-dict `:` type($source) `,` type($index) `,` type($result)";
   let hasVerifier = 0;
 }
+
```

Comment:
There are overlapping between this PR and the https://github.com/Xilinx/mlir-aie/pull/684. Just curious, do we intend to merge both PRs?

Reply:
Good catch! It shouldn't be. In my local repository this branch hangs from the other, and the PR includes both by default (my bad).